protostar/commands/test/test_command.py

```diff
@@ -162,6 +158,12 @@ def arguments(self):
         ]
 
     async def run(self, args: Namespace) -> TestingSummary:
+        structured_format = False
+        if vars(args).get("json"):
+            structured_format = args.json
+        else:
+            args.json = None
```

Comment:
This code could be removed. You can use `isinstance(HumanMessenger)` to use the progress bar in the live logger.

Reply:
[use isinstance HumanMessenger instead of additional variable](https://github.com/software-mansion/protostar/pull/1254/commits/7b99db00b0fa7cb38b6323eab3a4c92525116de4)